deploy/ethereum-MKRUSD/README.md

```diff
@@ -0,0 +1,10 @@
+
+# MedianMKRUSD
+
+### Mainnet
+
+Median: [0xdbBe5e9B1dAa91430cF0772fCEbe53F6c6f137DF](https://etherscan.io/address/0xdbBe5e9B1dAa91430cF0772fCEbe53F6c6f137DF#code)
+
+### Testnet
+
+Median: [0x496C851B2A9567DfEeE0ACBf04365F3ba00Eb8dC](https://goerli.etherscan.io/address/https://goerli.etherscan.io/address/0x496C851B2A9567DfEeE0ACBf04365F3ba00Eb8dC#code)
```

Comment:
Link broken. Should be https://goerli.etherscan.io/address/0x496C851B2A9567DfEeE0ACBf04365F3ba00Eb8dC.

Reply:
thanks, fixored